CMakeLists.txt

```diff
@@ -252,6 +247,13 @@ else(HIOP_SPARSE)
   set(HIOP_USE_STRUMPACK OFF CACHE BOOL "Build without STRUMPACK" FORCE)
 endif(HIOP_SPARSE)
 
+# It simplifies linking against metis to set an internal variable
+if(HIOP_SPARSE OR HIOP_WITH_KRON_REDUCTION)
+  set(HIOP_USE_METIS ON CACHE BOOL "Link against METIS library ${METIS_LIBRARY}" FORCE)
```

Comment:
Do we need `FORCE` here?
```suggestion
  set(HIOP_USE_METIS ON CACHE BOOL "Link against METIS library ${METIS_LIBRARY}")
```

Reply:
Similar is done at several other places.

Reply:
Here I was just following the same configuration that was used for other TPLs such as COINHSL, and STRUMPACK (above). If I should re-configure accordingly then I will apply this to all the TPLs.

Reply:
This is probably out of this PR scope.